app/src/main/java/com/andela/helpmebuy/activities/PurchaseRequestActivity.java

```diff
@@ -121,16 +129,27 @@ private void sendRequest(PurchaseRequest purchaseRequest, String id) {
                 = new FirebaseCollection<PurchaseRequest>("purchaseRequest/" + id, PurchaseRequest.class);
         firebaseCollection.save(purchaseRequest, new DataCallback<PurchaseRequest>() {
             @Override
-            public void onSuccess(PurchaseRequest data) {}
+            public void onSuccess(PurchaseRequest data) {
+            }
 
             @Override
-            public void onError(String errorMessage) {}
+            public void onError(String errorMessage) {
+            }
         });
     }
 
-    @Override
-    public void onClick(View v) {
-        createPurchaseRequest();
-        finish();
+    public void showSnackBar() {
+        CoordinatorLayout coordinatorLayout = (CoordinatorLayout) findViewById(R.id.purchase_request);
+        Snackbar snackbar = Snackbar.make(coordinatorLayout, "Tap the button with the plus sign to add a new item", Snackbar.LENGTH_LONG);
+        snackbar.show();
+    }
+
+    public void displayText() {
+        String name = getIntent().getExtras().getString("name");
+        String[] firstName = name.split(" ");
+        if (firstName.length > 1) {
```

Comment:
Why check the length of firstName? what about if I only registered with just one name? I  think (in case we don't have) we should have different fields for first name and last name in the DB.